### PR TITLE
[PATCH v2] example: generator: use platform defined pktio_env

### PR DESCRIPTION
**Pull request by SunilKumarKori**
odp_generator uses hard coded null interface to validate functionality.
Patch optimises it to use platform defined pktio_env.

Signed-off-by: Sunil Kumar Kori <skori@marvell.com>

### CODE REVIEW

**Comment by SunilKumarKori** [comment]
@matiaselo @jerinjacobk 
Please review.

File: example/generator/generator_run.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (c) 2018, Linaro Limited
```
Comment (matiaselo) [inline example/generator/generator_run.sh:3]: This is basically a new file, so in my opinion you could change the copyright to `Copyright (c) 2020, Marvell`

**Comment by matiaselo** [comment]
@SunilKumarKori I had one comment but otherwise the PR is looking good. You can add my reviewed-by tags.

**Comment by SunilKumarKori** [comment]
> @SunilKumarKori I had one comment but otherwise the PR is looking good. You can add my reviewed-by tags.

Update the PR after changing the Copyright.

**Comment by SunilKumarKori** [comment]
@matiaselo Changes are updated as per suggestion.  I think, its ready to be merged.

**Comment by SunilKumarKori** [comment]
@matiaselo I think, Its ready to be merged. Can you please do the needful if it looks okay to you.

**Review by matiaselo — APPROVED** [review]

**Comment by matiaselo** [comment]
>I think, Its ready to be merged. Can you please do the needful if it looks okay to you.

@SunilKumarKori sure, I'll merge the PR.

**Pull request merged**